src/protocols/LoRaWAN/LoRaWAN.cpp

```diff
@@ -1061,29 +1262,29 @@ bool LoRaWANNode::verifyMIC(uint8_t* msg, size_t len, uint8_t* key) {
 int16_t LoRaWANNode::setPhyProperties() {
   // set the physical layer configuration
   int16_t state = RADIOLIB_ERR_NONE;
-  if(this->FSK) {
-    // for FSK, configure the channel
-    state = this->phyLayer->setFrequency(this->band->fskFreq);
-    RADIOLIB_ASSERT(state);
-    DataRate_t dr;
-    dr.fsk.bitRate = 50;
-    dr.fsk.freqDev = 25;
-    state = this->phyLayer->setDataRate(dr);
-    RADIOLIB_ASSERT(state);
-    state = this->phyLayer->setDataShaping(RADIOLIB_SHAPING_1_0);
-    RADIOLIB_ASSERT(state);
-    state = this->phyLayer->setEncoding(RADIOLIB_ENCODING_WHITENING);
-  }
-  RADIOLIB_ASSERT(state);
+  // if(this->FSK) {
+  //   // for FSK, configure the channel
+  //   state = this->phyLayer->setFrequency(this->band->fskFreq);
+  //   RADIOLIB_ASSERT(state);
+  //   DataRate_t dr;
+  //   dr.fsk.bitRate = 50;
+  //   dr.fsk.freqDev = 25;
+  //   state = this->phyLayer->setDataRate(dr);
+  //   RADIOLIB_ASSERT(state);
+  //   state = this->phyLayer->setDataShaping(RADIOLIB_SHAPING_1_0);
+  //   RADIOLIB_ASSERT(state);
+  //   state = this->phyLayer->setEncoding(RADIOLIB_ENCODING_WHITENING);
+  // }
+  // RADIOLIB_ASSERT(state);
   
```

Comment:
Is there a specific reason this is disabled? I can't seem to find the FSK configuration elsewhere.

Reply:
Re-integrated; had to ignore for the band system rework